test/integration/taxon_browsing_test.rb

```diff
@@ -0,0 +1,164 @@
+require 'integration_test_helper'
+require 'slimmer/test_helpers/govuk_components'
+
+class TaxonBrowsingTest < ActionDispatch::IntegrationTest
+  include RummagerHelpers
+  include TaxonHelpers
+  include Slimmer::TestHelpers::GovukComponents
+
+  def search_results
+    [
+      {
+        'title' => 'Content item 1',
+        'description' => 'Description of content item 1',
+        'link' => 'content-item-1'
+      },
+      {
+        'title' => 'Content item 2',
+        'description' => 'Description of content item 2',
+        'link' => 'content-item-2'
+      },
+    ]
+  end
+
+  describe 'Browsing taxon with grandchildren' do
```

Comment:
Have you considered using the cucumber-method-style from content-tagger integration tests for this?